### PR TITLE
Keychain: adds support for private keys as non-array

### PR DESCRIPTION
**Pull request by macor161**
# 🦅 Pull Request Description

- Adds support for a single key (non-array) from `mainnet_key.json` or `rinkeby_key.json` file.

## Rational

Multiple users have reported that the Buidler plugin didn't work after switching from aragonCLI. It turns out they had a key file that contained a single key, not in an array.

### CODE REVIEW

File: src/config/mnemonic.ts [hunk 1]
```diff
@@ -46,7 +47,9 @@ export const configExtender: ConfigExtender = (finalConfig, userConfig) => {
       const { rpc, keys } = byNetworkMnemonic
       if (!finalNetwork.url && rpc) finalNetwork.url = rpc
       if (!finalNetwork.accounts && keys)
-        finalNetwork.accounts = ensureHexEncoding(keys)
+        finalNetwork.accounts = isArray(keys)
```
Comment (sohkai) [inline src/config/mnemonic.ts:50]: Ah, rather than importing this from lodash, we can just use `Array.isArray()`.

Comment (macor161) [reply src/config/mnemonic.ts]: Good point!

**Comment by macor161** [comment]
> We might need to update their package.jsons/yarn.locks, in a similar manner to aragon/aragon#1434.

Yes this is exactly the problem.

> But are we not installing the tests via yarn? The lock files should be preventing errors.

No we use a pretty old version of the client, which doesn't have any `yarn.lock` or `package-lock.json` unfortunately: commit [775edd606333a111eb2693df53900039722a95dc](https://github.com/aragon/aragon/tree/775edd606333a111eb2693df53900039722a95dc).
Until [issue 1368](https://github.com/aragon/aragon/issues/1368) is fixed, I could perhaps push a git tag that contains a patch for the babel problem.

**Comment by sohkai** [comment]
Ohhh gotcha; this is also due to the client?

I thought the tests were having trouble building each app's frontend.

**Comment by dapplion** [comment]
Could we pulled the build client from somewhere (e.g. IPFS or Github releases) instead of building it locally?

**Comment by macor161** [comment]
@dapplion Yes I think the idea was to move to Github releases but we currently can't use them as they contain a [bug](https://github.com/aragon/aragon/issues/1368) in development. When the issue is fixed, I definitely think it would be a good idea.

**Comment by dapplion** [comment]
I'm a firm believer that the client should not have to be built multiple times. Configuration should be handled at another layer, which would allow reusing the content in APM for all possible needs

**Comment by sohkai** [comment]
@macor161 I've merged this branch with master, now that we have the hotfix in place.

------

> I'm a firm believer that the client should not have to be built multiple times. Configuration should be handled at another layer, which would allow reusing the content in APM for all possible needs

Definitely agree; I've made https://github.com/aragon/buidler-aragon/issues/175 for this separately.

**Pull request merged**